resources/js/components/auth/LoginComponent.vue

```diff
@@ -1,189 +1,184 @@
-                                    </div>
-                                </div>
-                            </div>
+                    <label class="form-check-label" for="remember"
+                      >Remember Me
+                    </label>
+                  </div>
+                </div>
+              </div>
 
-                            <div class="form-group row mb-0">
-                                <label
-                                    class="col-md-4 col-form-label text-md-right"
-                                ></label>
-                                <div class="col-md-8 offset-md-4">
-                                    <button
-                                        type="submit"
-                                        class="btn btn-primary"
-                                    >
-                                        Login
-                                    </button>
+              <div class="form-group row mb-0">
+                <label class="col-md-4 col-form-label text-md-right"></label>
+                <div class="col-md-8 offset-md-4">
+                  <button type="submit" class="btn btn-primary">Login</button>
 
-                                    <a
-                                        class="btn btn-link"
-                                        href="/forgot-password"
-                                        >Forgot Your Password?</a
-                                    >
-                                </div>
-                            </div>
-                        </form>
-                    </div>
+                  <a class="btn btn-link" href="/forgot-password"
+                    >Forgot Your Password?</a
+                  >
                 </div>
-            </div>
+              </div>
+            </form>
+          </div>
         </div>
+      </div>
     </div>
+  </div>
 </template>
 
 <script>
+import { mapState } from "vuex";
 export default {
-    data() {
-        return {
-            email: "",
-            password: "",
-            remember: "",
-            errors: {}
-        };
-    },
-    methods: {
-        login() {
-            let that = this;
-            this.axios
-                .post("/login", {
-                    email: this.email,
-                    password: this.password,
-                    remember: this.remember
-                })
-                .then(function(response) {
-                    // alert(response);
-                    window.location.href = "/task";
-                })
-                .catch(function(error) {
-                    that.errors = error.response.data.errors;
-                })
-                .finally(() => (this.loading = false));
-        }
+  data() {
+    return {
+      email: "",
+      password: "",
+      remember: "",
+    };
+  },
+  computed: {
+    ...mapState("login", ["errors"]),
+  },
+  methods: {
+    login() {
+      this.$store.dispatch("login/login", {
```

Comment:
Login and Register are two actions for the Auth. So instead of creating them as separate modules, create a single module called `auth` and put `login` and `register` actions in it.

Reply:
ok